tests/app/questionnaire/routing/test_operators.py

```diff
@@ -0,0 +1,291 @@
+from datetime import datetime, timedelta
+
+import pytest
+from dateutil.relativedelta import relativedelta
+
+from app.questionnaire.routing.helpers import datetime_as_midnight
+from app.questionnaire.routing.operator import Operator
+from app.questionnaire.rules import convert_to_datetime
+
+now = datetime.utcnow()
+now_as_yyyy_mm_dd = now.strftime("%Y-%m-%d")
+now_as_yyyy_mm = now.strftime("%Y-%m")
+now_as_yyyy = now.strftime("%Y")
+
+equals_operations = [
```

Comment:
This would be easy if ordered, either by type (string, dates, bool etc) my preference, or result (True/False) it currently looks random to my eyes and difficult to see the gaps

Reply:
Forgot to order when I got rid of some tests. I went with ordering by result, then by type.